packages/react-lowcode/src/codegen/generation/generators/detail/mui-detail-generator.ts

```diff
@@ -140,6 +140,33 @@ export default class MuiDetailGenerator implements DetailGenerator {
                             astChanged = true;
                           }
                         }
+                        else if (inputProp.value !== prop.initializer.expression.getText()) {
```

Comment:
It will be good to put here comment with example of expression for which this should work. Please put this logic into separate function.